### PR TITLE
Slider

### PR DESCRIPTION
**Pull request by marinanicolai**

### CODE REVIEW

File: src/components/ImageItem/index.js [hunk 1]
```diff
@@ -1,52 +1,16 @@
...
+import { ImageContainer, Image } from "./ImageItem.styles";
 
+const Item = ({ item, index, likes, setCurrentImageIndex }) => {
   return (
-    <div>
-      {/* <Modal isOpen={isOpen} onRequestClose={toggleModal}> */}
-      <img
+    <ImageContainer>
+      <Image
         src={item.urls["small"]}
```
Comment (yoloOnTheBattlefield) [inline src/components/ImageItem/index.js:8]: item.urls.small

File: src/components/PhotoGallery/PhotoGallery.styles.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+export const Profile = styled.div`
+  background-image: url(${(props) => props.image});
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 31px;
+  width: 31px;
+  cursor: pointer;
+`;
```
Comment (yoloOnTheBattlefield) [inline src/components/PhotoGallery/PhotoGallery.styles.js:40]: SavedImg, Likes and Profile have the same code, consider reusing it

File: src/components/PhotoGallery/index.js [hunk 3]
```diff
@@ -1,12 +1,124 @@
...
+      onClick={() => setCurrentImageIndex(currentImageIndex - 1)}
+    />
+  );
+
+  const RightArrow = () => (
+    <Arrow
+      image={rightArrow}
+      onClick={() => setCurrentImageIndex(currentImageIndex + 1)}
+    />
+  );
```
Comment (yoloOnTheBattlefield) [inline src/components/PhotoGallery/index.js:54]: These two arrows, seem overkill, can you reuse the same function?

File: src/components/Slider/index.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
+import React, { useState, useRef, useEffect } from "react";
+import Slider from "react-slick";
+import { Box } from "./Silder.styles";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
```
Comment (yoloOnTheBattlefield) [inline src/components/Slider/index.js:5]: I think it's better to import these in the main index.js file

File: src/components/Slider/index.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  const [nav1, setNav1] = useState(null);
+  const [nav2, setNav2] = useState(null);
+  const slider1 = useRef(null);
+  const slider2 = useRef(null);
+
+  useEffect(() => {
+    setNav1(slider1.current);
+    setNav2(slider2.current);
+  }, []);
+  console.log(data);
```
Comment (yoloOnTheBattlefield) [inline src/components/Slider/index.js:18]: no console

File: src/components/Slider/index.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+          <div>
+            <h3>5</h3>
+          </div>
+          <div>
+            <h3>6</h3>
+          </div>
+        </Slider>
+
+        <div>
+          {/* <ImageItem key={data?.[0]?.id} /> */}
```
Comment (yoloOnTheBattlefield) [inline src/components/Slider/index.js:44]: no commented out code

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/PhotoGallery/PhotoGallery.styles.js (hunk 2 above)
Comment (marinanicolai) [reply src/components/PhotoGallery/PhotoGallery.styles.js:40]: I didn't finish styling the items, so it will change in the future.

**Pull request merged**